VimeoUpload/Upload/Extensions/NSError+Upload.swift

```diff
@@ -67,14 +56,9 @@ public enum UploadErrorDomain: String
 
 public enum UploadLocalErrorCode: Int
 {
-    case cannotEvaluateDailyQuota = 0 // "User object did not contain uploadQuota.quota information"
-    case cannotCalculateDiskSpace = 1 // "File system information did not contain NSFileSystemFreeSize key:value pair"
-    case cannotEvaluateWeeklyQuota = 2 // "User object did not contain uploadQuota.space information"
-    
-    case diskSpaceException = 3
-    case assetIsNotExportable = 4    
-    case dailyQuotaException = 5
-    case weeklyQuotaException = 6
+    case cannotCalculateDiskSpace // "File system information did not contain NSFileSystemFreeSize key:value pair"
+    case diskSpaceException
+    case assetIsNotExportable
```

Comment:
@nicolelehrer It looks like these were only used for local error matching which makes sense since integers don't mean anything without context in Localytics.